CONTRIBUTING.md

```diff
@@ -19,9 +19,8 @@ The user experience captured in the OLM V1 PRD introduces many requirements that
 - [The Operator-Controller project](https://github.com/operator-framework/operator-controller/), which is the top level component allowing users to specify operators they'd like to install.
 - [The Catalogd project](https://github.com/operator-framework/catalogd/), which hosts operator content and helps users discover installable content.
 - [The Deppy project](https://github.com/operator-framework/deppy/), which enables the operator-controller to identify valid installs and upgrades from the list of installable content provided via the Catalogd project.
-- [The RukPak project](https://github.com/operator-framework/rukpak/), which facilitates the installation of operators.
```

Comment:
This technically still has a dependency on rukpak as a library right? It might be worth it to still mention rukpak since this is the case.

Reply:
@varshaprasad96 any thoughts on restoring this mention?

Reply:
Rukpak till now has been mentioned as a separate entity/API controller to manage installation. I would prefer removing this as users should not be concerned where our helpers reside. We are not intending to support the APIs provided in Rukpak separately, so having it not mentioned is good.

Reply:
This is in the `CONTRIBUTING.md` where those interested in contributing would go. I think it is reasonable to mention rukpak if we are intending for libraries/helpers to be maintained there long term.